data/json/items/comestibles/alcohol.json

```
@@ -1345,5 +1345,32 @@
     "phase": "liquid",
     "flags": [ "UNSAFE_CONSUME", "EATEN_COLD" ],
     "fun": 20
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "sake",
+    "name": { "str_sp": "sake" },
+    "weight": "36 g",
+    "color": "light_red",
+    "addiction_type": "alcohol",
+    "use_action": "ALCOHOL",
+    "stim": -4,
+    "container": "bottle_glass",
+    "comestible_type": "DRINK",
+    "symbol": "~",
+    "quench": 10,
+    "healthy": -2,
+    "addiction_potential": 4,
+    "calories": 204,
+    "description": "The traditional alcohol of Japan, this wine is made from rice.",
+    "price": "550 cent",
+    "price_postapoc": "1 USD",
+    "material": [ "alcohol", "water" ],
+    "primary_material": "alcohol",
+    "volume": "250 ml",
+    "phase": "liquid",
+    "charges": 7,
+    "flags": [ "UNSAFE_CONSUME", "EATEN_COLD" ],
```

```suggestion
    "flags": [ "UNSAFE_CONSUME", "EATEN_HOT", "EATEN_COLD" ],
```
Enjoy sake warm or cold both!